### PR TITLE
Create a new intent to support forgot password by email by phone number input

### PR DESCRIPTION
**Pull request by tung2744**

### CODE REVIEW

File: pkg/latte/intent_login.go [hunk 1]
```diff
@@ -148,7 +149,30 @@ func (i *IntentLogin) GetEffects(ctx context.Context, deps *workflow.Dependencie
...
 func (i *IntentLogin) OutputData(ctx context.Context, deps *workflow.Dependencies, w *workflow.Workflow) (interface{}, error) {
-	return nil, nil
+	verifiedAuthns, err := i.getVerifiedAuthenticators(w)
+	if err != nil {
+		return nil, err
+	}
+	if len(verifiedAuthns) == 0 {
+		return nil, nil
+	}
+	userID := verifiedAuthns[0].UserID
```
Comment (louischan-oursky) [inline pkg/latte/intent_login.go:159]: Why do we just use `i.Identity.UserID`?

File: pkg/latte/intent_login.go [hunk 2]
```diff
@@ -148,7 +149,30 @@ func (i *IntentLogin) GetEffects(ctx context.Context, deps *workflow.Dependencie
...
+	identities, err := deps.Identities.ListByUser(userID)
```
Comment (louischan-oursky) [inline pkg/latte/intent_login.go:160]: I think we can just get the standard attribute `email` and mask it, include it in the data as `masked_email`.

File: pkg/latte/intent_login.go (hunk 1 above)
Comment (tung2744) [reply pkg/latte/intent_login.go:159]: I think they are the same, but we should check `len(verifiedAuthns)` to ensure the user passed at least one step before including the email in response.

File: pkg/latte/intent_login.go (hunk 2 above)
Comment (tung2744) [reply pkg/latte/intent_login.go:160]: I am not sure if it is safe to assume the email in standard attribute is same as the email identity. As forgot password only works for login ids, I get them from login ids.

**Comment by tung2744** [comment]
Added a commit to expose ui params in sms data.

**Pull request merged**